pkgs/by-name/an/ani-skip/package.nix

```
@@ -0,0 +1,54 @@
+{
+  stdenvNoCC,
+  fetchFromGitHub,
+  makeWrapper,
+  gnugrep,
+  gnused,
+  curl,
+  fzf,
+  lib,
+}:
+stdenvNoCC.mkDerivation (final: {
+  pname = "ani-skip";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "synacktraa";
+    repo = "ani-skip";
+    rev = final.version;
+    hash = "sha256-VEEG3d6rwTAS7/+gBKHFKIg9zFfBu5eBOu6Z23621gM=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+  runtimeInputs = [
+    gnugrep
+    gnused
+    curl
+    fzf
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -D skip.lua $out/share/mpv/scripts/skip.lua
+    install -Dm 755 ani-skip $out/bin/ani-skip
+
+    runHook postInstall
+  '';
+
+  postFixup = ''
+    sed -i "s,\(--script-opts=%s\),--script=$out/share/mpv/scripts/skip.lua \1," $out/bin/ani-skip
+
+    wrapProgram $out/bin/ani-skip \
+      --suffix PATH : ${lib.makeBinPath final.runtimeInputs}
+  '';
+
+  meta = {
+    homepage = "https://github.com/synacktraa/ani-skip";
+    description = "An automated solution to bypassing anime opening and ending sequences.";
```

```suggestion
    description = "Automated solution to bypassing anime opening and ending sequences";
```

Contribution guidelines dictate no indefinite articles (e.g. "A ..." or "An ...") and no period at the end.